paper-fab.html

```diff
@@ -103,12 +108,14 @@
       iron-icon {
         @apply(--paper-fab-iron-icon);
       }
-      
+
       span {
         width: 100%;
         white-space: nowrap;
         overflow: hidden;
         text-overflow: ellipsis;
+        text-align: center;
+        @apply(--paper-fab-label);
```

Comment:
Also note that many properties that you might want to set to style the text (e.g. `color`, `font`, `text-transform`) inherit, so you can just set those on `paper-button` and it'll apply to the label.